cmp/viewmanager/ViewManagerModel.ts

```diff
@@ -123,6 +123,12 @@ export interface ViewManagerConfig {
      * Optional user-facing display name for describing global views. Defaults to 'global'
      */
     globalDisplayName?: string;
+
+    /**
+     * Optional key to pass a method that returns customized BlueprintJS `menuItem` for listing
+     * views in the ViewManager menu.
+     */
+    customViewMenuItem?: (view: ViewInfo, model: ViewManagerModel) => ReactNode;
```

Comment:
how about calling this thing thing `viewMenuItemFn`.  Custom is implied.

Reply:
Sure! Done.